machiavelli-cards/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Extra logging related to initialization of Log4j.
+ Set to debug or trace if log4j initialization is failing. -->
+<Configuration>
+    <Appenders>
+        <RollingFile name="App-File-Appender"
+                     fileName="application.log"
+                     filePattern="application.%d{yyyy-MM-dd}.log"
+                     ignoreExceptions="false">
+            <PatternLayout>
+                <pattern>
+                    %d{HH:mm:ss.SSS} %-5level %class{36} %L %M - %msg%xEx%n
+                </pattern>
+            </PatternLayout>
+            <CronTriggeringPolicy schedule="0 0 0 * * ?" />
+            <DefaultRolloverStrategy>
+                <Delete basePath="${sys:log-path}" maxDepth="1">
+                    <IfFileName glob="*/application.*.log" />
+                    <IfLastModified age="30d" />
+                </Delete>
+            </DefaultRolloverStrategy>
+        </RollingFile>
+        <!-- Console appender configuration -->
+        <Console name="Console-Appender" target="SYSTEM_OUT">
+            <PatternLayout>
+                <pattern>
+                    %d{HH:mm:ss.SSS} %-5level %class{36} %L %M - %msg%xEx%n
+                </pattern>
+            </PatternLayout>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <logger name="org.hibernate.SQL" additivity="false" level="debug">
+            <appender-ref ref="App-File-Appender" />
+            <appender-ref ref="Console-Appender" />
+        </logger>
+        <logger name="org.hibernate.type" additivity="false" level="debug">
+            <appender-ref ref="App-File-Appender" />
+            <appender-ref ref="Console-Appender" />
+        </logger>
+        <logger name="org.hibernate.type" additivity="false" level="debug">
+            <appender-ref ref="App-File-Appender" />
+            <appender-ref ref="Console-Appender" />
+        </logger>
+        <logger name="pl.jellytech.machiavelli.cards" additivity="false" level="info">
```

Comment:
avoid this since it might have issues and info logs won't actually go to the console appender
In general on the server ONLY keep app-file appender

Console-appender we can add locally only

Reply:
now this will only get logged to console-appender :) 
The deployed version on github should have the App-File-Appender everywhere (only)
Locally you can have within your git shelf f.ex. a version where you add the console-appender to the root logger and set it to debug